### ISSUE TITLE
Multi-line text wrap

### ISSUE THREAD

**Issue by dromer** [sebpiq/pd-fileutils#44]
Comments in pd patches are rendered on a single line, even though the text-box has line-breaks.

Work-around is to specifically create separate text-boxes for each line, but this is very tedious (and writing docs is already a lot of effort).

Ideally pd-fileutils takes into account multi-line `text` objects and renders them accordingly.

**Comment by sebpiq** [comment]
@dromer for info, the continuing work on this project will happen there : https://github.com/sebpiq/WebPd_pd-renderer

**Comment by dromer** [comment]
I see, I didn't find this project yet :)

What exactly is the difference, is it a rewrite?

**Comment by sebpiq** [comment]
Yes that's a complete rewrite ... which is going much slower than I wish it would :cry: 

[All the code is split in small modules](https://github.com/sebpiq?tab=repositories&q=WebPd_&type=&language=&sort=)

Which will eventually just be imported in the WebPd and pd-fileutils libraries.

**Comment by dromer** [comment]
Ok! hope you find some time to make progress on this. :)

**Comment by giuliomoro** [comment]
@dromer https://github.com/sebpiq/pd-fileutils/pull/88

**Comment by dromer** [comment]
@giuliomoro cool! It's been a long time since I looked at this.

I thought that this project was basically deprecated in favor of https://github.com/sebpiq/WebPd_pd-renderer

Will need to give your fork a try when I have time (not guaranteed soon), but nice that can still see some development :)

**Comment by giuliomoro** [comment]
Maybe the other works better but I don't  see how to make it work, it doesn't  have any docs. Any suggestion on getting started?

**Comment by dromer** [comment]
I'm not using either of these projects atm.

Better use the issues tab on WebPd_pd-renderer for its support..

**Comment by sebpiq** [comment]
Hello @dromer !

Sorry I didn't see all your messages and PR fast enough !
This library is actually deprecated indeed. 
The parser has been replaced by https://github.com/sebpiq/WebPd_pd-parser
The renderer by https://github.com/sebpiq/WebPd_pd-renderer although I haven't touched the code of the renderer in a long while because I have been focusing on WebPd, and the renderer is not needed for it. It should be easy enough to pick back up though.
The pd-parser should be in pretty good shape, although it definitely needs work and welcomes any contribution (same for pd-renderer) !!!
